lighthouse-core/gather/driver.js

```diff
@@ -895,6 +917,13 @@ class Driver {
     // No timeout needed for Page.navigate. See #6413.
     const waitforPageNavigateCmd = this._innerSendCommand('Page.navigate', {url});
 
+    try {
+      await this.sendCommand('Security.enable');
+      await this.waitForSecurityIssuesCheck();
+    } finally {
+      this.sendCommand('Security.disable');
+    }
```

Comment:
To do this right, I think @paulirish is right and this should move into `waitForFullyLoaded`. We shouldn't be mucking around looking at network traffic before our traffic monitor is allowed to start monitoring :)

Take a look at `this._waitForLoadEvent` or `this._waitForNetworkIdle` for the `promise`/`cancel` cancellable promise model used, and `_waitForFullyLoaded` itself for where to put it in parallel with the timeout. It should be a pretty simple translation from `waitForSecurityIssuesCheck` already in this PR.

(and forgive the state of the code in those methods...they're super old and have pretty much only grown through accretion, not really refactored, so they're a bit creaky and brittle :)

Reply:
what about the `_waitForFrameNavigated` case?

Reply:
> what about the `_waitForFrameNavigated` case?

that's only used for loading `about:blank`, so we're ok there. We should probably make that clearer, though really if you're saying `waitForNavigated: true`, you're basically forgoing all our checks already.